### PR TITLE
docs: Add intro/ installing/ initializing to Flutter reference docs.

### PR DESCRIPTION
**Pull request by dshukertjr**
Currently, some menu items are missing from Flutter docs. This PR adds them. Also removed the typescript menu item for Flutter docs. 

Also remove Misc from Flutter docs for now.

### CODE REVIEW

**Comment by MildTomato** [comment]
@dshukertjr amazing 🔥 🔥

**Comment by dshukertjr** [comment]
I am getting some errors running locally. Looking into it now.

Screen Shot 2022-12-13 at 2 25 24

**Comment by dshukertjr** [comment]
I had left some imports from old code. It should be all good now!

File: spec/common-client-libs-sections.json [hunk 1]
```diff
@@ -13,12 +13,19 @@
...
+    "type": "markdown",
+    "libs": ["dart", "dart_v0"]
+  },
   {
     "title": "Typescript Support",
     "id": "typescript-support",
     "slug": "typescript-support",
     "type": "markdown",
-    "libs": ["js", "js_v1", "dart", "dart_v0"]
+    "libs": ["js", "js_v1"]
```
Comment (dshukertjr) [inline spec/common-client-libs-sections.json:28]: Hmm, `Typescript Support` still appears in the side bar. Any idea how I could remove it from Dart docs?
Screen Shot 2022-12-13 at 2 54 58

Comment (MildTomato) [reply spec/common-client-libs-sections.json:28]: ahh yea, ill mention it to Terry. it's a bug

File: spec/common-client-libs-sections.json [hunk 2]
```diff
@@ -13,12 +13,19 @@
     "type": "markdown",
     "libs": ["js", "js_v1", "dart", "dart_v0"]
   },
+  {
+    "title": "Initializing",
+    "id": "initializing",
+    "slug": "initializing",
+    "type": "markdown",
+    "libs": ["dart", "dart_v0"]
+  },
```
[the remark this answers is not in the document]
Comment (MildTomato) [reply spec/common-client-libs-sections.json:22]: same as above. we are not checking wether they exist

File: spec/common-client-libs-sections.json (hunk 1 above)
Comment (dshukertjr) [reply spec/common-client-libs-sections.json:28]: Cool!

I will add some other Flutter docs specific fixes mentioned in slack to this PR while this bug is being worked on!

File: spec/common-client-libs-sections.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (MildTomato) [reply spec/common-client-libs-sections.json:22]: we'll merge anyway. separate issue

File: spec/common-client-libs-sections.json (hunk 1 above)
Comment (MildTomato) [reply spec/common-client-libs-sections.json:28]: we'll merge anyway. separate issue

Comment (dshukertjr) [reply spec/common-client-libs-sections.json:28]: Sounds good! I'm good to merge this one right now!

Comment (dshukertjr) [reply spec/common-client-libs-sections.json:28]: @MildTomato

**Review by MildTomato — APPROVED** [review]

**Pull request merged**